xml/installation-installation-configure_lbaas.xml

```diff
@@ -180,22 +180,28 @@ openssl req -x509 -passin pass:foobar -new -nodes -key cakey.pem -out serverca_0
    <emphasis role="bold">Installing the Amphora Image</emphasis>
   </para>
   <para>
-   Octavia is utilizing &o_comp; VMs for its load balancing function and &vendor;
+   Octavia uses &o_comp; VMs for its load balancing function and &vendor;
    provides images used to boot those VMs called
-   <literal>octavia-amphora-haproxy</literal>.
+   <literal>octavia-amphora</literal>.
   </para>
   <warning>
    <para>
     Without these images the Octavia load balancer will not work.
    </para>
   </warning>
+  <para vendor="suse">
+   The &ostack; load balancing service (Octavia) does not automatically upload
+   the Amphora guest image. The image
+   <filename>/srv/www/suse-12.3/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image.rpm</filename>
```

Comment:
If I understand this correctly, this image exists on some local machine already -- but is not clear to me what machine it is coming from..?

Also, if this is available locally on SOC, why would it not be available locally on HOS?